src/pykoala/cubing.py

```diff
@@ -570,29 +570,32 @@ class Cube(SpectraContainer):
     n_rows = None
     x_size_arcsec = None
     y_size_arcsec = None
-
-    def __init__(self, hdul=None, file_path=None, 
-                 hdul_extensions_map=None, **kwargs):
+    hdul_extensions_map = {"INTENSITY": "INTENSITY", "VARIANCE": "VARIANCE"}
+    
+    def __init__(self, hdul=None, hdul_extensions_map=None, **kwargs):
 
         self.hdul = hdul
-        self.hdul_extensions_map = hdul_extensions_map
 
-        if self.hdul_extensions_map is None:
-            self.hdul_extensions_map = {"INTENSITY": "INTENSITY",
-                                        "VARIANCE": "VARIANCE"}
-        if self.hdul is not None:
-            print("[Cube] Initialising cube from input HDUL")
-            self.hdul = hdul
-        elif file_path is not None:
-            self.load_hdul(file_path)
-        self.get_wcs_from_header()
+        if hdul_extensions_map is not None:
+            self.hdul_extensions_map = hdul_extensions_map
```

Comment:
I'm not sure whether `hdul_extensions_map` should be a class or an instance variable

Reply:
a new class variable `default_hdul_extension_map`  is replacing the previous one.